frontend/src/__tests__/ResetPasswordPage.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { ThemeProvider, theme } from '@chakra-ui/core'
+import { MemoryRouter } from 'react-router-dom'
+import { render, waitFor, fireEvent } from '@testing-library/react'
+import { I18nProvider } from '@lingui/react'
+import { setupI18n } from '@lingui/core'
+import { MockedProvider } from '@apollo/react-testing'
+import { UserStateProvider } from '../UserState'
+import ResetPasswordPage from '../ResetPasswordPage'
+import { UPDATE_PASSWORD } from '../graphql/mutations'
+
+const mocks = [
+  {
+    request: {
+      query: UPDATE_PASSWORD,
+    },
+    result: {
+      data: {
+        status: 'string',
+      },
+    },
+  },
+]
+
+describe('<CreateUserPage />', () => {
+  describe('given no input', () => {
+    describe('when onBlur fires', () => {
+      describe('password field', () => {
+        it('displays an error message', async () => {
+          const { container, queryByText } = render(
+            <UserStateProvider
+              initialState={{ userName: null, jwt: null, tfa: null }}
+            >
+              <ThemeProvider theme={theme}>
+                <I18nProvider i18n={setupI18n()}>
+                  <MemoryRouter initialEntries={['/']} initialIndex={0}>
+                    <MockedProvider mocks={mocks}>
+                      <ResetPasswordPage />
```

Comment:
You'll need to wrap this with a Route [like in this test](https://github.com/canada-ca/tracker/blob/5d814925aa206529e60a0b1056213187c237eec1/frontend/src/__tests__/DmarcGuidancePage.test.js#L56-L63), to ensure that useParams returns something.